db/migrate/20150717142559_add_distributions_to_repository.rb

```diff
@@ -0,0 +1,9 @@
+class AddDistributionsToRepository < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :distribution_version, :string
+    add_column :katello_repositories, :distribution_arch, :string
```

Comment:
Any reason to give this a relationship to the Architectures model instead of its own column?